Source/WebKit/Shared/Cocoa/AuxiliaryProcessCocoa.mm

```diff
@@ -343,4 +347,34 @@ static void disableDowngradeToLayoutManager()
 #endif
 }
 
+void AuxiliaryProcess::increaseFileDescriptorLimit()
+{
+    struct rlimit currentLimits = { };
+    if (int returnCode = getrlimit(RLIMIT_NOFILE, &currentLimits)) {
+        RELEASE_LOG_ERROR(Process, "Could not getrlimit(RLIMIT_NOFILE): %d", returnCode);
+        return;
+    }
+
+    int mib[] = { CTL_KERN, KERN_MAXFILESPERPROC };
+    rlim_t maxFilesPerProc = 0;
+    size_t len = sizeof(maxFilesPerProc);
+    if (int returnCode = sysctl(mib, 2, &maxFilesPerProc, &len, NULL, 0)) {
```

Comment:
The comments suggest that `maxFilesPerProc` should be an `int`, not an `rlim_t`, which is `uint64_t`.